docs/en/integrations/data-ingestion/kafka/msk/index.md

```diff
@@ -61,3 +61,14 @@ consumer.max.partition.fetch.bytes=1048576
 
 You can find more details (both implementation and other considerations) in the official [Kafka](https://kafka.apache.org/documentation/#consumerconfigs) and 
 [Amazon MSK](https://docs.aws.amazon.com/msk/latest/developerguide/msk-connect-workers.html#msk-connect-create-custom-worker-config) documentation.
+
+## Notes on Networking for MSK Connect
+
+In order for MSK Connect to connect to ClickHouse, your MSK cluster must be in a private subnet with a Private NAT connected for internet access. Instructions on how to set this up are provided below.
```

Comment:
> your MSK cluster must be in a private subnet with a Private NAT connected for internet access

Is it a strict requirement for MSK Connect? What if the customer's cluster is public and is in a public subnet?

Reply:
It is essentially a strict requirement, if the customer's cluster is in a public subnet it will not be assigned an elastic IP (Unless the user explicitly assigns one and associates it with it's ENI which I could never get to work, and [AWS describes as a workaround that is not recommended](https://docs.aws.amazon.com/msk/latest/developerguide/msk-connect-internet-access.html)) which means it won't be able to connect to the clickhouse cluster without having an IP address.

Reply:
This doc explains it well - let's link it in our documentation

Reply:
Updated wording to a recommendation and added the link to the AWS docs